Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -522,12 +526,11 @@ void GcodeSuite::G33() {
       #define Z0(I) ZP(0, I)
 
       // calculate factors
-      const float cr_old = delta_calibration_radius;
-      if (_7p_9_center) delta_calibration_radius *= 0.9f;
+      if (_7p_9_center) calibration_radius_factor = 0.9f;
```

Comment:
Must be dcr *= 0.9f to limit the calibration ratio if 3 circles are probed at 1.1*dcr, 1.0*dcr and 0.9*dcr